app/controllers/api/v1/topics_controller.rb

```diff
@@ -0,0 +1,12 @@
+module Api
+  module V1
+    class TopicsController < ApplicationController
+      def index
+        @topics = policy_scope(Topic).includes(image_attachment: :blob)
+        respond_to do |format|
+          format.json
+        end
```

Comment:
If you are only expecting to return json views you can do something like

```suggestion
        render json: @topics
```

Reply:
Enzo this is how postman do the GET /api/v1/topics
"topics": [
        {
            "id": 1,
            "name": "Futbol",
            "image": {
                "name": "image",
                "record": {
                    "id": 1,
                    "name": "Futbol",
                    "created_at": "2023-05-24T13:17:24.665Z",
                    "updated_at": "2023-05-24T13:17:24.691Z"
                }
            },
            "created_at": "2023-05-24T13:17:24.665Z",
            "updated_at": "2023-05-24T13:17:24.691Z"
        }

And this is where i call the @topics controller
json.topics do |json|
  json.array! @topics, :id, :name, :image, :created_at, :updated_at
end
So i resolve the comment without using the render.